charts/karpenter/templates/clusterrole.yaml

```diff
@@ -29,6 +29,9 @@ metadata:
   {{- end }}
 rules:
   # Read
+  - apiGroups: ["discovery.k8s.io"]
```

Comment:
Is it possible to constraint this by resource name or namespace? I'm assuming that this permission might be too wide right now since it's cluster-wide.

Reply:
Scoped it down by resource name. To constrain it by namespace, I think I'd need to create a new role in the default namespace for karpenter. Thoughts?